packages/dds/tree/package.json

```diff
@@ -86,6 +86,7 @@
 		"test": "npm run test:mocha",
 		"test:benchmark:report": "mocha --exit --perfMode --parentProcess --fgrep @Benchmark --reporter @fluid-tools/benchmark/dist/MochaReporter.js --timeout 60000",
 		"test:coverage": "c8 npm test",
+		"test:customBenchmarks": "mocha --config ./.mocharc.customBenchmarks.cjs",
```

Comment:
small nit: `this-kind-of-syntax` seems to be more consistent instead of camelCase

Reply:
https://github.com/microsoft/FluidFramework/pull/22140#discussion_r1712671076

I was first thinking `this:kind:of:syntax`. But applied @alexvy86 's suggestion.